test.sh

```diff
@@ -2,12 +2,12 @@
 [[ -e test.sh ]] || { echo >&2 "Please cd into the script location before running it."; exit 1; }
 set -e
 ./build.sh
-dotnet test -f netcoreapp1.0 MoreLinq.Test
-RID=`dotnet --info | grep RID | sed -E 's/ *RID: *(.+)/\1/'`
+dotnet exec MoreLinq.Test/bin/Debug/netcoreapp1.0/MoreLinq.Test.dll
```

Comment:
Sad that this is needed 😞 , do you know if this is expected to be fixed sometime?